help/en/html/doc/Technical/Names.shtml

```diff
@@ -70,20 +69,19 @@
         <li>Aspects for signals</li>
         <li>Blocks (track segments)</li>
         <li>Conditionals (for logiX, although these are not entirely named beans)</li>
-        <li>signal mast Groups</li>
-        <li>signal Heads</li>
-        <li>signal masts (F, for some reason)</li>
```

Comment:
The funny capitalization on these lines was intentional: it shows the letter representing the type: signal Head is H, etc

Reply:
The fuzzy capitalization here did not match the fuzzy capitalization in the by-type-letter list later; nor did it match what was implemented (prior to this PR) in code. I though it better to have one list that that matches the code and can be considered canonical instead of two lists in disagreement.

Reply:
After sleeping on it, I've removed this list entirely, instead referring readers to a following list.

Reply:
I think that's a great solution, thanks